src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,96 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api/api';
+import { Person } from '../../types';
+import { getVisiblePeople } from '../../utils/visiblePeople';
+import { Loader } from '../Loader';
+import { PeopleInfo } from '../PeopleInfo/PeopleInfo';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[] | []>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  const { slug } = useParams<{ slug: string }>();
+
+  const loadPeople = async () => {
+    setIsLoading(true);
+
+    try {
+      const loadedPeople = await getPeople();
+
+      setPeople(loadedPeople);
+      setIsLoaded(true);
+    } catch (error) {
+      setHasError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    loadPeople();
+  }, []);
+
+  const visiblePeople = getVisiblePeople(people);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+
+          {hasError && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+          {visiblePeople.length === 0 && !isLoading && (
```

Comment:
I guess would be better if the variable for that was kept separately. When you have multiple conditions in conditional rendering - it looks kinda difficult to read
```suggestion
          {!visiblePeople.length && !isLoading && (
```